src/macaron/slsa_analyzer/checks/provenance_verified_check.py

```diff
@@ -92,8 +92,9 @@ def run_check(self, ctx: AnalyzeContext) -> CheckResultData:
                 result_type=CheckResultType.FAILED,
             )
 
-        if not (build_type and build_type == "https://github.com/slsa-framework/slsa-github-generator/generic@v1"):
-            # Provenance is verified.
+        if not build_type == "https://github.com/slsa-framework/slsa-github-generator/generic@v1":
```

Comment:
I wonder if the purpose of this change is for simplifying the if statement, or it's because it's failing the integration test :thinking:

Reply:
It was intended as a simplification. Also improved in [348ebc7](https://github.com/oracle/macaron/pull/791/commits/348ebc7bb8385e77338bc7f57466ea3799b000d9)